hexrd/ui/image_mode_widget.py

```diff
@@ -79,41 +81,50 @@ def all_widgets(self):
 
         return widgets
 
-    def block_widgets(self):
-        previous = []
-        for widget in self.all_widgets():
-            previous.append(widget.blockSignals(True))
+    def update_gui_from_config(self):
+        blocked = [QSignalBlocker(x) for x in self.all_widgets()]  # noqa: F841
+        self.ui.cartesian_pixel_size.setValue(
+            HexrdConfig().cartesian_pixel_size)
+        self.ui.cartesian_virtual_plane_distance.setValue(
+            HexrdConfig().cartesian_virtual_plane_distance)
+        self.ui.cartesian_plane_normal_rotate_x.setValue(
+            HexrdConfig().cartesian_plane_normal_rotate_x)
+        self.ui.cartesian_plane_normal_rotate_y.setValue(
+            HexrdConfig().cartesian_plane_normal_rotate_y)
+        self.ui.polar_pixel_size_tth.setValue(
+            HexrdConfig().polar_pixel_size_tth)
+        self.ui.polar_pixel_size_eta.setValue(
+            HexrdConfig().polar_pixel_size_eta)
+        self.ui.polar_res_tth_min.setValue(
+            HexrdConfig().polar_res_tth_min)
+        self.ui.polar_res_tth_max.setValue(
+            HexrdConfig().polar_res_tth_max)
+        self.ui.polar_apply_snip1d.setChecked(
+            HexrdConfig().polar_apply_snip1d)
+        self.ui.polar_snip1d_width.setValue(
+            HexrdConfig().polar_snip1d_width)
+        self.ui.polar_snip1d_numiter.setValue(
+            HexrdConfig().polar_snip1d_numiter)
+
+    def auto_generate_cartesian_params(self):
```

Comment:
This is fine for the default setting (_i.e._, using the z component alone) with the plane normal anti-parallel to the beam direction; not if the plane is oblique (would need to use norm ar something).  But for generating the *initial* parameters for the view upon load, the logic makes sense.